build-system/tasks/check-types.js

```diff
@@ -35,7 +36,54 @@ const {typecheckNewServer} = require('../server/typescript-compile');
  * Note: This is a TEMPORARY holding point during the transition to type-safety.
  * @type {!Array<string>}
  */
-const PRIDE_FILES_GLOBS = ['src/resolved-promise.js', 'src/types.js'];
+const PRIDE_FILES_GLOBS = [
+  // Core
+  'src/core/**/*.js',
+
+  // Polyfills
+  'src/polyfills/abort-controller.js',
+  'src/polyfills/abort-controller.js',
+  'src/polyfills/array-includes.js',
+  'src/polyfills/document-contains.js',
+  'src/polyfills/domtokenlist.js',
+  'src/polyfills/map-set.js',
+  'src/polyfills/math-sign.js',
+  'src/polyfills/object-assign.js',
+  'src/polyfills/object-values.js',
+  'src/polyfills/set-add.js',
+  'src/polyfills/string-starts-with.js',
+  'src/polyfills/weakmap-set.js',
```

Comment:
Any reason to to use `src/polyfills/*.js`?

Reply:
There are a couple I'm missing since they depend on services